app/js/questions_controller.js

```diff
@@ -114,6 +109,17 @@
           show_more = true;
         }
 
+        var date_format = new Intl.DateTimeFormat(navigator.language, {
+          weekday: 'long', month: 'long', day: 'numeric'
+        });
```

Comment:
This works great and is super useful but, is it supported in FxOS 1.3 or is that not a concern anymore? I see it is Gecko 29, so it is probably fine. Should we perhaps use the l10n_date lib instead?